packages/api/src/routes/medical/network.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import {
+  getHieDirectoryEntriesByFilter,
+  getHieDirectoryEntriesByFilterCount,
+} from "../../external/carequality/command/cq-directory/get-cq-directory-entries";
+import { requestLogger } from "../helpers/request-logger";
+import { asyncHandler } from "../util";
+import { dtoFromHieDirectoryEntry, NetworkDTO } from "./dtos/networkDTO";
+import { networkGetSchema } from "./schemas/network";
+
+import { PaginatedResponse } from "@metriport/shared";
```

Comment:
nit: spaces prevent auto-organize imports from doing its thing 😄

Reply:
oh thats why my auto import organizer wasn't working? XD I was confused how this got bye it!